samples/can/prj.conf.mcp2515

```diff
@@ -0,0 +1,19 @@
+# CAN + MCP2515 config
```

Comment:
you could split that into another patch.

Reply:
Done

Reply:
Done